src/entry.js

```diff
@@ -123,7 +123,7 @@ module.exports = argv => {
     process.exit(1);
   }
 
-  commandFunc.apply(commands, [context].concat(args)).catch(err => {
+  commandFunc(context, ...args).catch(err => {
```

Comment:
Left this `.catch` so we didn't grab any unintended `Error`s here.

Reply:
i was worried about this change but it looks like it didn't need to be passing the command object before, so we're good!